htdocs/js/smr15.js

```diff
@@ -102,8 +102,78 @@ function showRaceInfo(select) {
 	var desc = document.getElementById('race_descr');
 	$(desc).children().addClass('hide');
 	$(".race_descr" + race_id, desc).removeClass('hide');
+	createRaceRadarChart(race_id);
 }
 
+// Used by game_join.php 
+function createRaceRadarChart(race_id) {
+
+	var races = {
+		1: [[1.0,  1.0,  1.0,  1.0,  1.0],  'Neutral',    '#FFD800'],
```

Comment:
Since we're using an associative array now, there's no reason to have an entry for "Neutral" anymore. Let's just remove it.